tests/e2e/steps/steps_def.go

```diff
@@ -884,28 +883,48 @@ func configureAuthorizationProxyServer(cr csmv1.ContainerStorageModule) error {
 		driverNamespace = ""
 	)
 
+	//by default, use set defined in env file
+	endpointvar := "END_POINT"
+	systemIdvar := "SYSTEM_ID"
+	uservar := "STORAGE_USER"
+	passvar := "STORAGE_PASSWORD"
+	poolvar := "STORAGE_POOL"
+
+	// if tests are running multiple scenarios that require differently configured auth servers, we will not be able to use one set of vars
+	// this section is for powerflex, other drivers can add their sections as required.
+	if driver == "powerflex" {
```

Comment:
Will this change break powerscale with authorization related scenarios?

Reply:
when driver isn't powerflex, it'll default to using the env-file variables as it did before:
```	endpointvar := "END_POINT"
	systemIdvar := "SYSTEM_ID"
	uservar := "STORAGE_USER"
	passvar := "STORAGE_PASSWORD"
	poolvar := "STORAGE_POOL"````
so other drivers still work - I also verified auth + pscale scenarios still work